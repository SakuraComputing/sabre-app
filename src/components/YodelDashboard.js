import React from 'react';

const YodelDashboard = () => (
    <div></div>
);
export default YodelDashboard;
