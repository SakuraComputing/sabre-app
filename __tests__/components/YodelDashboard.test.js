import React from 'react';
import YodelDashboard from '../../src/components/YodelDashboard';
import { shallow } from 'enzyme';

describe('Yodel Dashboard',() => {
   it('should render correctly',() => {
     const wrapper = shallow(<YodelDashboard/>);
     expect(wrapper).toMatchSnapshot();
   });
});

