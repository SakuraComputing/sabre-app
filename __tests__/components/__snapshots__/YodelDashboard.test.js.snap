// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Yodel Dashboard should render correctly 1`] = `<div />`;
